### PR TITLE
CB-20436 Add more logging to find long transactions and jdbc statements

### PR DESCRIPTION
**Pull request by sodre90**
See detailed description in the commit message.

### CODE REVIEW

File: common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateNPlusOneCircuitBreaker.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import com.sequenceiq.cloudbreak.app.StaticApplicationContext;
 
-public class HibernateNPlusOneCircuitBreaker extends HibernateNPlusOneLogger {
+public class HibernateNPlusOneCircuitBreaker extends HibernateSatementStatisticsLogger {
```
Comment (bergerdenes) [inline common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateNPlusOneCircuitBreaker.java:8]: typo: `HibernateStatementStatisticsLogger`

File: common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateSatementStatisticsLogger.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package com.sequenceiq.cloudbreak.common.tx;
+
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.cloudbreak.app.StaticApplicationContext;
+
+public class HibernateSatementStatisticsLogger extends HibernateStatementStatistics {
```
Comment (bergerdenes) [inline common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateSatementStatisticsLogger.java:10]: typo: `HibernateStatementStatisticsLogger`

File: common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateNPlusOneCircuitBreaker.java (hunk 1 above)
Comment (sodre90) [reply common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateNPlusOneCircuitBreaker.java]: Thanks! Fixed

File: common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateTransactionInterceptor.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        this.maxTransactionTimeThreshold = StaticApplicationContext.getEnvironmentProperty("hibernate.transaction.warning.max.time",
+                Long.class, DEFAULT_TRANSACTION_MAX_TIME_WARNING);
+    }
+
+    @Override
+    public void afterTransactionCompletion(Transaction tx) {
+        long txCompletionTime = System.nanoTime();
+        LOGGER.trace("Transaction {} completed at {} nanosec", tx, txCompletionTime);
+        Long txStartTime = transactionMap.get(tx);
+        transactionMap.remove(tx);
```
Comment (lturcsanyi) [inline common/src/main/java/com/sequenceiq/cloudbreak/common/tx/HibernateTransactionInterceptor.java:35]: map.remove returns the value (or null), get is unnecessary

**Review by lturcsanyi — APPROVED** [review]

**Pull request merged**